host-configs/rzansel-blueos_3_ppc64le_ib_p9-clang@8.0.0_nvcc_xlf.cmake

```diff
@@ -110,6 +110,7 @@ set(AXOM_CUDA_ARCH "sm_70" CACHE PATH "")
 
 set(CMAKE_CUDA_FLAGS "-restrict -arch ${AXOM_CUDA_ARCH} -std=c++11 --expt-extended-lambda -G" CACHE PATH "")
 
+set(CUDA_SEPARABLE_COMPILATION ON CACHE BOOL "" )
```

Comment:
This is going to be lost when we regenerate the host-configs..... It needs to be added to scripts/uberenv/packages/uberenv-axom/package.py

Reply:
Thanks @white238 -- I forgot to point that out.

Would you mind adding it to your PR -- https://github.com/LLNL/axom/pull/122 ?
Or should we do this separately, e.g. in @bmhan12's branch?

It should go in here:
https://github.com/LLNL/axom/blob/134082dfad44a90b1a90ecb5b4e3e9be0bc1bc65/scripts/uberenv/packages/uberenv-axom/package.py#L427-L449